.github/workflows/review_ci.yml

```diff
@@ -9,18 +9,14 @@ on:
 jobs:
   reviewdog:
     name: Run reviewdog
-    runs-on: [ self-hosted ]
-    container: 
-      image: c.rzp.io/proxy_dockerhub/golang:1.16.5-alpine3.13
-      credentials:
-        username: ${{ secrets.HARBOR_DOCKER_USERNAME }}
-        password: ${{ secrets.HARBOR_DOCKER_PASSWORD }}
-      volumes:
-        - /etc/hosts:/etc/hosts
+    runs-on: ubuntu-latest
+    container: golang:1.16.5-alpine3.13
```

Comment:
Upgrade to v1.22 or v1.23

Reply:
This can has to be taken by owner of apps. As devops we are not aware of its function testing.